### PR TITLE
Sig 1931 trigger tel and email software keyboard

### PR DESCRIPTION
**Pull request by jasperswart**
in this PR:
- in meldingsformulier added type of phone and email address fields
- meldingsformulier is now using Input component
- removed snapshot test from meldingsformulier TextInput
- tested in :

iPhone 13: safari
iPad (sim) 13: safari
iPhone 8 plus: safari

Android
Galaxy S10: chrome, firefox and samsung browser
Galaxy S9: chrome, firefox and samsung browser
Pixel 4: chrome, firefox
Pixel 2: chrome, firefox
Galaxy Note 10: chrome, firefox, samsung browser

Windows Phone: Lumia 930

### CODE REVIEW

File: src/signals/incident/definitions/wizard-step-3-telefoon.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ export default {
           subtitle: 'Zo kunt u ons helpen het probleem sneller of beter op te lossen.',
           path: 'reporter.phone',
           placeholder: 'Telefoonnummer',
-          type: 'text',
+          type: 'tel',
           autoRemove: /[^ ()\d+-]/g,
```
Comment (janjaap) [inline src/signals/incident/definitions/wizard-step-3-telefoon.js]: What if someone would input their phone number like +31612345678? Would that result in `31612345678`?
The regex seems a bit strange to me; does it even do anything?

**Comment by janjaap** [comment]
There might be other fields that could benefit form a change in type; I saw, for instance,
- `extra_container_number`
- `extra_personen_overig`
- `extra_straatverlichting_niet_op_kaart_nummer`
- `extra_klok_niet_op_kaart_nummer`
- `extra_verkeerslicht_nummer`

File: src/signals/incident/definitions/wizard-step-3-telefoon.js (hunk 1 above)
Comment (jasperswart) [reply src/signals/incident/definitions/wizard-step-3-telefoon.js]: on blur it will remove unwanted characters: "+" is allowed

Comment (janjaap) [reply src/signals/incident/definitions/wizard-step-3-telefoon.js]: The `autoRemove` option doesn't seem to work. Can you confirm?

Comment (jasperswart) [reply src/signals/incident/definitions/wizard-step-3-telefoon.js]: again: on blur the invalid characters are removed. please look at telefoonnummer step

https://acc.meldingen.amsterdam.nl/incident/telefoon

**Pull request merged**